.github/ISSUE_TEMPLATE/unified-release.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+name: Unified Release Manager
+description: Test
+labels:
+  - unified-release
+
+title: Unified Release
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Unified Release Day
+
+      We are releasing a new version today.
+      Please make sure that the automation complete.
+
+- type: input
+  id: upstream
+  attributes:
+    label: Parent issue
```

Comment:
There are pros/cons in both cases.
In my opinion, we could keep the parent relationship between those issues and let the PR reference the project issue to avoid confusion in the upstream issue.